### DISCUSSION TITLE
My XrmTool is still in pending status

### DISCUSSION THREAD

**Discussion by durgaprasadkatari** [MscrmTools/XrmToolBox#1028] [Q&A]
Hi, 
I have published the package into nuget and created new tool registration request in XrmTools website. But the tool is still pending. Is there any manual process to approve the tool.

**Comment by MscrmTools** [comment]
Hi, absolutely, the process is manual. I will check, sorry for the delay

**Comment by durgaprasadkatari** [comment]
Ok, could you please approve it when you have time.

**Comment by durgaprasadkatari** [comment]
Could you please let me know when it is published.

**Comment by durgaprasadkatari** [comment]
Hi, I have updated the author info and removed the sample notification in the tool. I have released the new version to the NuGet package. Could you please review and approve it now?
By the way, I can't reproduce the null reference error on Generating the register XML file.

**Comment by MscrmTools** [comment]
Here is how I reproduce the error
https://www.screencast.com/t/KuJqIjywhP

**Comment by durgaprasadkatari** [comment]
Ok thanks for the information. I have resolved the null reference error now and published the new version into nuget. Could you please check and let me know.

**Comment by durgaprasadkatari** [comment]
Sorry if you are busy. Would you mind approving my tool?

**Comment by MscrmTools** [comment]
@durgaprasadkatari can you be patient? I have a job, XrmToolBox is not what makes me live...

**Comment by durgaprasadkatari** [comment]
Ok, no problem. I'm sorry to bother you.

**Comment by BetimBeja** [comment]
@MscrmTools can you move this to Discussions! I see no issue on XrmToolBox, and nothing community related actually.

**Reply by MscrmTools** [reply]
Thank you for asking! Done!

**Comment by durgaprasadkatari** [comment]
@MscrmTools I have fixed the below issues and published the new version to NuGet.
- The tool does not handle correctly unavailable IOrganizationService.
- When clicked directly on "Generate Registration File" and it was not asking the user to select where to save the file.
- Updated UI a little bit.

**Reply by MscrmTools** [reply]
I'm really doing your basic testing work here...
When user clicks on "Register Plugins" button, your code should check if the files are valid. I can click on this button without setting any registration file and/or assembly dll. And your code crash because the file "Brows Registration File" does not exist which is normal.
Please fix this and your tool should be good to be validated

**Reply by durgaprasadkatari** [reply]
I'm really sorry, I had included this validation logic in my first version. But recently when I changed the UI this logic somehow was not working. I have fixed it now and pushed the new version into nuget.
Hopefully this is the last time I bother you. Thanks!
